lib/publiccloud/instance.pm

```diff
@@ -264,6 +264,33 @@ sub upload_log {
     assert_script_run("test -d '$tmpdir' && rm -rf '$tmpdir'");
 }
 
+=head2 verify_upload_logs
+
+    verify_upload_logs();
+
+Check files status and upload logs ok to UI.
+=cut
+
+sub verify_upload_logs {
+    my ($self, $dir, @files) = @_;
+    my @logs = ();
+    my @missing_logs = ();
+
+    # Select existing files
+    foreach (@files) {
+        my $res = $self->ssh_script_run(cmd => "test -e $dir/" . $_, quiet => 1, proceed_on_failure => 1);
+        $res = isok($res) ? push(@logs, $_) : push(@missing_logs, $_);
+    }
+    record_info("Logs missing", "Files not found in $dir: " . join(', ', @missing_logs)) if ($#missing_logs > 0);
+
+    # Upload existing logs to openqa  UI
+    if ($#logs > 0) {
+        $self->ssh_script_run(cmd => 'sudo chmod a+r ' . join(' ', map { "$dir/$_" } @logs), proceed_on_failure => 1);
+        $self->upload_log("$dir/" . $_, log_name => 'measure_boottime_' . $_ . '.txt', failok => 1) foreach (@logs);
```

Comment:
Why using `measure_bootime_` as prefix?

Reply:
Inherited from previous edxisting code and style, that you can see in any test logs section: the routine name leading the log, helping to know where it came from.

Reply:
> Inherited from previous existing code and style, that you can see in any test logs section: the routine name always added before the log name, helping to know where it came from.

But now you intend it to be a library function.  The name would be confusing.

Reply:
Yes, I shall change that name, to be generalized.

Reply:
Header input removed, standard module name applied